src/scanner.coffee.md

```diff
@@ -139,14 +139,15 @@ of cells or null if the scanner is exhausted.
         return callback() if response and response.statusCode is 204
         return callback err if err
         cells = []
+        encoding = if @options.encoding is 'undefined' then @options.encoding else @client.options.encoding
```

Comment:
This seems like a mistake, moreover options.encoding should equal an undefined string